source/class/qx/Promise.js

```diff
@@ -903,7 +902,12 @@ qx.Class.define("qx.Promise", {
         return;
       }
       qx.Promise.__initialized = true;
-      qx.bom.Event.addNativeListener(window, "unhandledrejection", qx.Promise.__onUnhandledRejection.bind(this));
+      var isNode = typeof process !== "undefined";
+      if (isNode) {
+        process.on("unhandledRejection", qx.Promise.__onUnhandledRejection.bind(this));
```

Comment:
Maybe check whether the user has already registered an unhandled rejection handler? Or remove this one when they do register one?

Reply:
This is old framework code. Only adopted to node.

Reply:
Still an issue, but ok.